module/rds.py

```diff
@@ -136,165 +178,1292 @@
-      default: null
-    maint_window:
+      default: null      
+    preferred_backup_period:
       description:
-        - "Maintenance window in format of ddd:hh24:mi-ddd:hh24:mi.  (Example: Mon:22:00-Mon:23:15) If not specified then a random maintenance window is assigned."
+        - Backup period.
       required: false
       default: null
-    backup_window:
+      aliases: ['backup_window']
+    backup_retention_period:
       description:
-        - Backup window in format of hh24:mi-hh24:mi.  If not specified then a random backup window is assigned.
+        - "Number of days backups are retained.  Set to 0 to disable backups.  Default is 7 day.  Valid range: 7-730.
       required: false
       default: null
-    backup_retention:
+      aliases: ['backup_retention']
+    db_tags:
       description:
-        - "Number of days backups are retained.  Set to 0 to disable backups.  Default is 1 day.  Valid range: 0-35.
+        - A hash/dictionaries of db tags, '{"tag_key": "tag_value"}', tag_key must be not null when tag_value isn't null.
       required: false
       default: null
+    wait:
+      description:
+        - wait for the RDS instance to be in state 'running' before returning.
+      required: false
+      default: "no"
+      choices: [ "yes", "no" ]
+    wait_timeout:
+      description:
+        - how long before wait gives up, in seconds
+      required: false
+      default: 300
 
+function: change rds instance type
+  description: change rds instance type
+  command: modify
+  options:
+    instance_id:
+      description:
+        - ID of the database to change.
+      required: true
+      default: null      
+    region:
+      description:
+        - The ACS region to use. If not specified then the value of the ECS_REGION environment variable,
+        if any, is used.
+      required: false
+      aliases: [ 'acs_region', 'ecs_region' ]
+    db_instance_class:
+      description:
+        - The instance type of the database.  Must be specified when command=create. Optional when command=replicate,
+        command=modify. If not specified then the replica inherits the same instance type as the source instance.
+      required: false
+      default: null
+      aliases: ['instance_type']
+    db_instance_storage:
+      description:
+        - Size in gigabytes of the initial storage for the DB instance.
+      required: false
+      default: null
+      aliases: ['size']
+    pay_type:
+      description:
+        - The pay type of the DB instance.
+      required: true
+      default: null
+      choices: ['Postpaid']
 
-function: replicate db instance
+function: create an rds read-only instance
   description: create a read-only replica of an existing instance.
   command: replicate
   options:
     source_instance:
       description:
         - ID of the database to replicate.
-      required: false
+      required: true
       default: null
+      aliases: ['instance_id']
     region:

```

Comment:
In the line 275, please unify the parameter region with above 'alicloud_region'.